docs/photoniq/event-delivery/clients/index.md

```diff
@@ -1,23 +1,33 @@
 ---
 sidebar_position: 1
-title: Event Delivery Client SDKs
+title: Getting Started with Event Delivery Client SDKs
```

Comment:
> Apologies for the misleading branch name. I made some changes to VWRs.

There are changes to other parts of documentation, too. Please make create PRs for separate tasks in the future.